src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarJobDslExtension.java

```diff
@@ -43,19 +55,22 @@ public class CodeSonarJobDslExtension extends ContextExtensionPoint {
     @RequiresPlugin(id = "codesonar", minimumVersion = "2.0.0")
     @DslExtensionMethod(context = PublisherContext.class)
     public Object codesonar(
```

Comment:
...this of course assumes that we need to include the `@RequiresPlugin` decorator.  It seems unnecessary since it would be impossible to read this particular class method without having the required plugin already installed.  It seems more likely that `@RequiresPlugin` is needed for declaring a dependency on an external plugin.

Even if we think it is unnecessary, it might be unwise to remove the decorator in this release, since we are (were??) trying to keep this release low-risk.